examples/Example/babel.config.js

```diff
@@ -8,6 +8,10 @@ module.exports = {
         extensions: ['.js', '.ts', '.tsx'],
         alias: {
           'react-native-gesture-handler': '../../',
+          'react-native-gesture-handler/DrawerLayout':
```

Comment:
Did the example work without it earlier? Or is it somehow related to this PR?

Reply:
It worked (in metro & TS) but eslint wasn't happy, since we use an import plugin that takes config from babel aliases.